academy/3-running-a-chain/node-api-and-cli.md

```diff
@@ -259,20 +255,20 @@ $ ./simd keys list
 or with:
 
 ```sh
-$ ./simd keys show b9lab
+$ ./simd keys show academy
 ```
 
 ## Make yourself a proper validator
 
-As you know by now, a Cosmos SDK blockchain relies on identified validators to produce blocks. Initially there is no validator to generate blocks. You are in a catch-22 situation: your initialized and unstarted chain needs a genesis account and validator for bootstrapping purposes.
+As you know by now, a Cosmos SDK blockchain relies on identified validators to produce blocks. Initially there is no validator to generate blocks.
```

Comment:
Here, I think the catch-22 comment helps understand why we bother with genesis account and transaction.
At least, for those who come from a PoW world, it would not register, and they would believe that we add these genesis elements just because this is an exercise and we want to show how those things are done if you wanted to do them. When in fact you **have** to do them, which is what the catch-22 comment drills down.

Reply:
@8bitpal please check this comment. PR is approved

Reply:
I think that makes sense, but would just rephrase it differently (and would exclude the word "identified"):

```suggestion
As you know by now, a Cosmos SDK blockchain relies on validators to produce blocks. Initially there is no validator to generate blocks. In order to get your chain up and running, you'll need to set up at least one validator in your genesis file, and allocate some tokens so that these can be staked and the chain can be started. This is where your tokens are initially minted, because you define how many tokens are going to be part of the genesis file.
```